### PR TITLE
Made localhost_apiserver configurable: port and storage filepath

### PR DESCRIPTION
**Pull request by mateuszmidor**
KNTK-181

### CODE REVIEW

File: apiv6/localhost_apiserver/main.go [hunk 1]
```diff
@@ -10,20 +10,29 @@
...
 
 	router := cloudexportstub.NewRouter(CloudExportAdminServiceApiController)
 
-	log.Fatal(http.ListenAndServe(":8080", router))
+	log.Printf("Server started, port %s, storage %s", port, storageFilePath)
+	log.Fatal(http.ListenAndServe(port, router))
+}
+
+func readPortStorage() (port string, storage string) {
+	flag.StringVar(&port, "port", ":8080", "Port number for the server to listen to")
```
Comment (Fenthick) [inline apiv6/localhost_apiserver/main.go:34]:
```suggestion
	flag.StringVar(&address, "address", ":8080", "TCP address for the server to listen to")
```
If only port is passed (`8080`), the server will fail to start, I believe.

**Review by Fenthick — APPROVED** [review]
One nitpick found, but approving anyway

**Review by lukaa12 — DISMISSED** [review]

File: apiv6/localhost_apiserver/main.go (hunk 1 above)
Comment (mateuszmidor) [reply apiv6/localhost_apiserver/main.go]: updated port -> address

Comment (Fenthick) [reply apiv6/localhost_apiserver/main.go]: Yes, exactly, ":8080" is address and "8080" is port, this is what I meant.

**Review by Fenthick — DISMISSED** [review]

**Review by Fenthick — APPROVED** [review]

**Review by lukaa12 — APPROVED** [review]

**Review by TedatKentik — APPROVED** [review]
Local host - API Server configurable - port/storage file path

**Pull request merged**